app/liquid_tags/codesandbox_tag.rb

```diff
@@ -1,15 +1,19 @@
 class CodesandboxTag < LiquidTagBase
   PARTIAL = "liquids/codesandbox".freeze
+  # rubocop:disable Layout/LineLength
+  REGISTRY_REGEXP = %r{https?://(?:www|app\.)?(?:codesandbox\.io/embed/)(?<video_id>[a-zA-Z0-9-]{0,60})(?:\?)?(?<options>\S+)?}
```

Comment:
Note: I did not include a complicated parsing of url parameters in the Main URL regex, opting instead to just capture the whole query with `\S+`. This is because we do validate each parameter passed in within the `parse_options` method.

Reply:
Non-blocking: we can make this a bit shorter by using a character class and leaving off the lower bound in `{0, 60}`, as it defaults to 0.

```suggestion
  REGISTRY_REGEXP = %r{https?://(?:www|app\.)?(?:codesandbox\.io/embed/)(?<video_id>[\w-]{,60})(?:\?)?(?<options>\S+)?}
```